api/src/main/java/org/pmiops/workbench/utils/mappers/FirecloudMapper.java

```diff
@@ -20,4 +21,9 @@ default WorkspaceAccessLevel fcAccessLevelToApiAccessLevel(RawlsWorkspaceAccessE
 
   @Named("apiToFcWorkspaceAccessLevel")
   RawlsWorkspaceAccessLevel apiToFcWorkspaceAccessLevel(WorkspaceAccessLevel accessLevel);
+
+  @ValueMapping(source = "DISCOVERER", target = "NO_ACCESS")
+  @ValueMapping(source = "APPLICATION", target = "NO_ACCESS")
+  @ValueMapping(source = "SUPPORT", target = "NO_ACCESS")
```

Comment:
I am not sure about the SUPPORT role in VWB, perhaps instead of NO_ACCESS we should map it to READER.